blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy
+title: "What’s up with the Scala MOOCs?"
```

Comment:
Hum, reading that title makes me think that you're trying to address voiced concerns about the MOOCs. Have you considered something like "News from the Scala MOOCs" or "What's new about the Scala MOOCs?".

Reply:
I’ve changed for “News from the Scala MOOCs”